docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -9,38 +9,75 @@ use, command-line utility software written in C and based on open-source libFTDI
 library.
 
 1. Clone the repository:
-```
-git clone git://git.tizen.org/tools/testlab/sd-mux
-```
-
-2. Follow the instruction included in sd-mux/README:
-```
-Requirements:
-  1. libftdi1 1.4 - development library
-  2. popt - development library
-  3. cmake - binary tool
-
-Build:
- - enter into project directory
- - create 'build' directory
- - enter into "build" directory
- - run 'cmake ..'
- - run 'make'
-
-Install:
- - enter into 'build' directory
- - run 'sudo make install' to install binary into '/usr/local/bin' (the default
-   one) directory
-
-Note:
-If you want to install files into different directory then add argument to cmake
-command: `cmake -DCMAKE_INSTALL_PREFIX=/usr ..`
-
-Then again run:
- make
- make install
-```
-3. Now the environment is ready to configure and control the SDWire.
+    ```
+    git clone git://git.tizen.org/tools/testlab/sd-mux
+    ```
+2. Check whether the installation requirements for sd-mux are met:
+    * libftdi1 1.4 development library is installed. To do this, open the 
+       terminal and type the following command:
+        ```
+        dpkg -L libftdi1-dev
+        ```
+        If the library is installed, after typing the above command you will see 
+        information about the paths to the library components. 
+    * popt development library is installed. To do this, open the terminal and
+       type the following command:
+        ```
+        dpkg -L libftdi1-dev
+        ```
+        If the library is installed, after typing the above command you will see 
+        information about the paths to the library components. 
+    * cmake binary tool is installed. To do this, open the terminal and type 
+       the following command:
+        ```
+        cmake --version
+        ```
+        If the tool is installed, after typing the above command you will see 
+        information about the installed on your computer cmake version. 
+    
+    If any above-mentioned requirements are not met - go to point 3. If they are
+    met - go to point 4. 
+
+3. Install missing libraries and/or tools:
+   * libftdi1 1.4 development library. To do this, open the terminal and type 
```

Comment:
@sulewskiprzemyslaw it's rendering incorrectly. Try to run local preview, described here: https://github.com/Dasharo/docs/blob/master/README.md and instead of `mkdocs build`, run `mkdocs serve` and open your browser with http://localhost:8000/

Reply:
ok, I have fixed it by adding one space mark